docs/3.tutorials/examples/advanced-xcc.md

```diff
@@ -290,9 +290,7 @@ In order to deploy the contract you will need to create a NEAR account.
 Go into the directory containing the smart contract (`cd contract-advanced-ts` or `cd contract-advanced-rs`), build and deploy it:
 
 ```bash
-cargo near build
```

Comment:
it's somewhat pointless to do `cargo near build` in the same sequence with `cargo near deploy`, as the latter will do a build step anyway